src/data/cityWork/categorySubsets.js

```diff
@@ -0,0 +1,255 @@
+// cribbed from https://github.com/cityofsomerville/Somerville_Data_Pipes/blob/178b6f39f26e5d7619c3ee7b70c966eae69cdfbc/constituent-services.R#L126-L130
+// feel free to modify!
+
```

Comment:
This theoretically pins to the `category` column in the Somerville_Services table -- though it's out of date (it's further cribbed from the old Somerville_Data_Pipes repo). We should add a `fixMe` to make this a centralized thing.